armi/reactor/tests/test_parameters.py

```diff
@@ -502,3 +511,86 @@ class MockPCChild(MockPC):
         pcc = MockPCChild()
         with self.assertRaises(AssertionError):
             pcc.whatever = 33
+
+
+class ParamCollectionWhere(unittest.TestCase):
+    """Tests for ParameterCollection.where."""
+
+    class ScopeParamCollection(parameters.ParameterCollection):
+        pDefs = parameters.ParameterDefinitionCollection()
+        with pDefs.createBuilder() as pb:
+            pb.defParam(
+                name="empty",
+                description="Bare",
+                location=None,
+                categories=None,
+                units="",
+            )
+            pb.defParam(
+                name="keff",
+                description="keff",
+                location=parameters.ParamLocation.VOLUME_INTEGRATED,
+                categories=[parameters.Category.neutronics],
+                units="",
+            )
+            pb.defParam(
+                name="cornerFlux",
+                description="corner flux",
+                location=parameters.ParamLocation.CORNERS,
+                categories=[
+                    parameters.Category.neutronics,
+                ],
+                units="",
+            )
+            pb.defParam(
+                name="edgeTemperature",
+                description="edge temperature",
+                location=parameters.ParamLocation.EDGES,
+                categories=[parameters.Category.thermalHydraulics],
+                units="",
+            )
+
+    pc: typing.ClassVar[parameters.ParameterCollection]
+
```

Comment:
```suggestion
```

Reply:
Up to you, but I believe this line is unneeded. The `setupClass()` method handles this for you.